src/integrationTest/groovy/wooga/gradle/unity/UnityTaskIntegrationSpec.groovy

```diff
@@ -71,8 +71,10 @@ abstract class UnityTaskIntegrationSpec<T extends UnityTask> extends UnityIntegr
 
         then:
         result.wasExecuted(subjectUnderTestName)
-        result.standardOutput.contains("Starting process 'command '")
-        value == result.standardOutput.contains(" $expectedCommandlineSwitch")
+        def taskLogBeginIndex = result.standardOutput.indexOf("Task :${subjectUnderTestName}")
+        def stdOut = result.standardOutput.substring(taskLogBeginIndex)
+        stdOut.contains("Starting process 'command '")
+        value == stdOut.contains(" $expectedCommandlineSwitch")
```

Comment:
Can we make this a utility method so we don't copy the nearly same line everywhere? I would also suggest to check for the Task finish/failed whatever state. Means a method that gets the log for a given task. I like this idea.

Reply:
I have this in some project (dotnet-sonarqube I guess) ready to be extracted into the commons test lib. I'd rather put that there, and then use it here.

Reply:
I figured out that the place where I had this before was this project 😅 , I adapted the existing implementation and used it here.